libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionView.java

```diff
@@ -221,19 +220,17 @@ public void onChanged(@Nullable Boolean isOffRoute) {
   public void update(RouteProgress routeProgress) {
```

Comment:
Why are we exposing this using `@SuppressWarnings("UnusedDeclaration")`? 🤔
> No usages found in All Places

Could you clarify how this would be used?

Reply:
This API is used when `InstructionView` is used without a `NavigationViewModel`.  If you wanted to inflate the view separate of `NavigationView`, you would use this API with a `ProgressChangeListener` to provide it data.

Reply:
Gotcha. Thanks for clarifying 👍